src/components/Sidebar/index.jsx

```diff
@@ -201,6 +224,7 @@ Sidebar.propTypes = {
   enableAnalyticsScreen: PropTypes.bool,
   onWidthChange: PropTypes.func,
   isMobile: PropTypes.bool,
+  enterpriseGroupsV1: PropTypes.bool,
```

Comment:
nit: Would help readability to have a name that's a bit more explicit that it is a feature flag, IMO

Reply:
I think it follows the same naming pattern as the other features, and you can see thats its being brought in from `enterpriseFeatures`